mods/d2k/rules/structures.yaml

```diff
@@ -23,6 +23,8 @@ concretea:
 	Building:
 		Footprint: xx xx
 		Dimensions: 2,2
+	Tooltip:
+		Name: Concret Slab
```

Comment:
I fixed it by changing the file and  using `git rebase -i HEAD~3` `pick` -> `fixup` command. 
Is there a better way?